apps/website/lib/llm/handler.ts

```
@@ -0,0 +1,119 @@
+      body,
+    });
+
+    const responseData = await response.json();
+
+    if (!response.ok) {
+      console.error(`API error:`, responseData);
+      return cors(
+        request,
+        new Response(
+          JSON.stringify({
+            error: `API error: ${eval(`responseData.${config.errorMessagePath}`) || "Unknown error"}`,
```

Biome (1.9.4)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)